### DISCUSSION TITLE
Looking for pixel breakpoints for Paste

### DISCUSSION THREAD

**Discussion by mjanssen2021** [twilio-labs/paste#1552] [Q&A] [answered]
I couldn't find exact guidance on what breakpoints exactly Paste is optimized for... is that a thing?

For responsive sites - other teams I worked on usually have 4-6 specific breakpoints identified for responsive web based products/websites to make sure we always design for those breakpoint width sizes... (ie 400px smallest / 1600 px largest , etc. 

Does paste have that documented anywhere or is that even something we define? I couldn't find anything in the PASTE documentation or the discussions here in Github.

**Comment by richbachman** [comment, marked as answer]
Hi @mjanssen2021. Good question. Paste comes with built in breakpoints. We just haven't documented those, mainly because console isn't quite responsive. Because of that, the current breakpoints may change in future if a more responsive format is needed. The breakpoints are: 

- 25rem (400px)
- 64rem (1024px)
- 77rem (1232px)

Paste also comes with a method to pass custom breakpoints at the theme level.

**Reply by mjanssen2021** [reply]
Thank you! This helps a ton.
